target/linux/x86/base-files/etc/board.d/02_network

```diff
@@ -31,9 +31,6 @@ cisco-mx100-hw)
 	ucidef_set_network_device_path "eth11" "pci0000:00/0000:00:01.1/0000:02:00.2"
 	ucidef_set_interfaces_lan_wan "mgmt eth2 eth3 eth4 eth5 eth6 eth7 eth8 eth9 eth10 eth11" "wan"
 	;;
-mellanox-technologies-ltd-msn2100)
-	ucidef_set_interface_lan "eth1 eth2 eth3 eth4 eth5 eth6 eth7 eth8 eth9 eth10 eth11 eth12 eth13 eth14 eth15 eth16"
-	;;
```

Comment:
This just adds a default bridge over all the LAN ports and makes it available in LuCI. What can you change with the NVIDIA Spectrum Platform which makes this not ueefull?
I am not familiar with the NVIDIA Spectrum Platform.

Reply:
I think it is better to keep a default automatically generated configuration, and users can assign configurations to interfaces as needed, fo openwrt is concerned, now it not be possible to implement port config like sonic. 
(https://github.com/sonic-net/SONiC/wiki/SAI)
When using openwrt on these devices, users sure have a console connection, so there is no need to worry about being unable to configure the system without a network.
Port Lanes function description: https://github.com/Mellanox/mlxsw/wiki/Switch-Port-Configuration#port-lanes